exercise-001/main.cpp

```diff
@@ -3,32 +3,48 @@
 
 #include "CLI/CLI.hpp"
 #include "config.h"
-
+#include "vector"
+#include "random"
```

Comment:
```suggestion
#include <random>
```

Better use `<>` instead of `""` for includes from the stdlib